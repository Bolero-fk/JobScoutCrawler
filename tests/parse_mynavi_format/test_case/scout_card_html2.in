<a class="not_text_decoration" href="/mypage/apply/confirm?id=00001&amp;ai_flag=true"><div class="job_offer_heading"><div class="job_title_area"><div class="job_tag_area"><div class="job_tag_browed">閲覧済</div><div class="job_tag_type">正社員</div></div><div class="job_title">タイトル2</div></div><div class="job_match_area"><svg xmlns="" width="66" height="66" viewBox="0, 0, 66, 66" transform="rotate(-90)"><circle class="job_base_circle" cx="33" cy="33" r="30" stroke="url(#baseColors)"></circle><circle class="job_match_circle" cx="33" cy="33" r="30" stroke="url(#linearColors)" style="--max:189;"></circle></svg><div class="job_match_progress_content">マッチ度<br><span class="job_match_number" data-num="100">28</span>%</div></div></div><div class="job_label_area"><div class="job_label">ラベル1</div><div class="job_label">ラベル2</div><div class="job_label">ラベル3</div><div class="job_label">ラベル4</div><div class="job_label">ラベル5</div><div class="job_label">ラベル6</div><div class="job_label">ラベル7</div><div class="job_label">ラベル8</div><div class="job_label">ラベル9</div></div><div class="job_appeal_area"><div class="job_appeal_img"><img src=""></div><div class="job_appeal_point">●アピール1
●アピール2
●アピール3
●アピール4</div></div><div class="job_description_area"><div class="job_description_wrap"><div class="job_description_mark"><img src=""></div><div class="job_description_text job_description_company">社名非公開（※社名は転職・就職相談にて開示）</div></div><div class="job_description_wrap"><div class="job_description_mark"><img src=""></div><div class="job_description_text job_description_location">●東京都豊島区／東京本社
●石川県金沢市／ｏｏ営業所
●神奈川県川崎市／ｏｏ営業所
●茨城県つくば市／ｏｏ営業所
●福岡県福岡市</div></div><div class="job_description_wrap"><div class="job_description_mark"><img src=""></div><div class="job_description_text job_description_occupation">ソフトウェアエンジニア,テストエンジニア,データサイエンティスト,フロントエンドエンジニア,プロジェクトマネージャー</div></div><div class="job_description_wrap"><div class="job_description_mark"><img src=""></div><div class="job_description_text job_description_salary">400万円～1000万円</div></div></div></a><div class="job_button_area"><button type="button" class="application_button"><span class="application_button_inner">応募する</span></button></div>