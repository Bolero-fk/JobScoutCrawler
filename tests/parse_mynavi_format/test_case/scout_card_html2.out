{"company_name": "社名非公開（※社名は転職・就職相談にて開示）", "min_salary": 400, "max_salary": 1000, "location": "東京都豊島区／東京本社 石川県金沢市／ｏｏ営業所 神奈川県川崎市／ｏｏ営業所 茨城県つくば市／ｏｏ営業所 福岡県福岡市", "using_lang": "- - -", "description": "タイトル2\n【ラベル1】【ラベル2】【ラベル3】【ラベル4】【ラベル5】【ラベル6】【ラベル7】【ラベル8】【ラベル9】\n●アピール1 ●アピール2 ●アピール3 ●アピール4\nソフトウェアエンジニア,テストエンジニア,データサイエンティスト,フロントエンドエンジニア,プロジェクトマネージャー", "recieve_date": "- - -", "limit_day": "- - -", "site_name": "mynavi"}